karma.conf.js

```diff
@@ -12,13 +12,7 @@ const staticConfig = {
   port: 9876,
   colors: true,
   autoWatch: true,
-  browsers: [
-    'ChromeHeadless'
-    // 'ChromeDebug'
-    // 'FirefoxDebug'
-    // 'VirtualBoxIE11Win7'
-    // 'VirtualBoxEdgeWin10'
-  ],
```

Comment:
I removed these to simplify the manual test runner operation.

**old:** change this value in-place

**new:**
```bash
BROWSER=ChromeDebug make test
```